ocaml/idl/datamodel.ml

```diff
@@ -6978,9 +6978,21 @@ let vm_guest_metrics =
       field ~qualifier:DynamicRO ~ty:(Map(String, String)) "os_version" "version of the OS";
       field ~qualifier:DynamicRO ~ty:(Map(String, String)) "PV_drivers_version" 
 	"version of the PV drivers";
-      field ~qualifier:DynamicRO ~ty:Bool ~in_oss_since:None "PV_drivers_up_to_date"
-	"true if the PV drivers appear to be up to date";
-
+      field ~qualifier:DynamicRO ~ty:Bool ~in_oss_since:None
+        ~lifecycle:[
+          Removed, rel_dundee, "Disabled in favour of the PV drivers auto-update"
```

Comment:
This is deprecated now, not removed. Suggested text:
"Deprecated in favour of network_paths_optimised and storage_paths_optimised."